src/utils/deckShuffler.js

```diff
@@ -0,0 +1,9 @@
+const RANDOMNESS = 0.4;
+
+function shuffleDeck(deck) {
+  const shuffledDeck = [...deck];
+  shuffledDeck.sort(() => Math.random() - RANDOMNESS);
+  return shuffledDeck;
```

Comment:
Legal que tu já entende que tu pode mutar a argumento que tu recebe na função.

Eu acho que tu pode fazer tudo direto em um linha apenas:
```suggestion
  return [...deck].sort(() => Math.random() - RANDOMNESS);
```

Reply:
Ok! :fist_oncoming: